.github/workflows/tests.yml

```diff
@@ -32,6 +32,14 @@ jobs:
             git clone --depth=1 https://github.com/mod-audio/mod-host.git ~/mod-host
             make -C ~/mod-host -j $(nproc)
           fi
+      - name: build test blocks
+        shell: bash
+        run: |
+          cd test-blocks
```

Comment:
there is no need to "cd" here, just make path argument to "make -C <path>" below